src/API/Plugin.php

```diff
@@ -19,6 +21,14 @@ class Plugin extends Client
     const SETTING_EDITABLE_KEY = 'editable';
     const SETTING_MODIFIED_DATE_KEY = 'modified_on';
 
+    /**
+     * @param IntegrationInterface $integration
+     */
+    public function __construct(IntegrationInterface $integration)
+    {
+        parent::__construct($integration);
+    }
```

Comment:
https://3v4l.org/oXBX0 never mind, must have been a bug in my impl.  Fat fingering the name of `__construct()` or something.